src/data-view.js

```diff
@@ -107,6 +107,18 @@ function VirtualizedTable({ dataTable, highlightFn, showPointsFn }) {
           className="tr"
         >
           {row.cells.map((cell) => {
+
+            const currentValue = cell.value;
+
+            if(currentValue == "false"){
+              cell.value = currentValue.charAt(0).toUpperCase()
+              + currentValue.slice(1);
```

Comment:
Nit: this is more readable as `cell.value = "False"`. I don't mind if you keep it as is, though.